pom.xml

```diff
@@ -114,8 +114,7 @@
     <plugin.frontend.version>1.12.1</plugin.frontend.version>
 
     <!-- common library versions -->
-    <slf4j.version>1.7.30</slf4j.version>
-    <log4j.version>1.2.17</log4j.version>
+    <slf4j.version>1.7.35</slf4j.version>
```

Comment:
why not [1.7.36](https://mvnrepository.com/artifact/org.slf4j/slf4j-reload4j/1.7.36)?

Reply:
causes a maven convergence issue - due to other dependencies of zeppelin using 1.7.35

I can try further changes but it could take a few days.

The key thing is to get agreement that keeping log4jv1 is not a good thing and that reload4j is a reasonable short term change to avoid using a completely insecure logging framework.

Reply:
thanks for the elaboration, that makes sense.